Real_Masters_all/aagovp.xml

```diff
@@ -38,7 +38,7 @@
   <archdesc level="recordgrp" audience="external" type="inventory" relatedencoding="MARC21">
     <did>
       <origination>
-        <corpname source="lcnaf" encodinganalog="110">Ann Arbor (Mich.)</corpname>
+        <geogname source="lcnaf" encodinganalog="110">Ann Arbor (Mich.)</geogname>
```

Comment:
I think this should maybe stay as a corpname -- In this case, it is not the geographic location of Ann Arbor that is the creator of the Ann Arbor government publications, but the corporate entity of Ann Arbor. Subjects cannot be associated with resources as creators, only agents.